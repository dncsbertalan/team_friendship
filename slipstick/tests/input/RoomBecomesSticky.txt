load testmap.txt
move Janitor_1 Room_9
move Janitor_1 Room_10
move Janitor_1 Room_4
move Student_1 Room_1
move Student_1 Room_2
move Student_1 Room_3
move Student_1 Room_9
move Student_1 Room_10
state -r Student_1
move Student_1 Room_9
move Student_1 Room_10
move Student_1 Room_9
move Student_1 Room_10
move Student_1 Room_9
move Student_1 Room_10
move Student_1 Room_9
move Student_1 Room_10
state -r Student_1
exit