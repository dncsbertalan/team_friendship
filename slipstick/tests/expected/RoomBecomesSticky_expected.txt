Successfully loaded testmap.txt
Moving Janitor_1 from Janitors'Room to Room_9
Moving Janitor_1 from Room_9 to Room_10
Moving Janitor_1 from Room_10 to Room_4
Moving Student_1 from MainHall to Room_1
Moving Student_1 from Room_1 to Room_2
Moving Student_1 from Room_2 to Room_3
Moving Student_1 from Room_3 to Room_9
Moving Student_1 from Room_9 to Room_10
Room_10 state: cleaned
Moving Student_1 from Room_10 to Room_9
Moving Student_1 from Room_9 to Room_10
Moving Student_1 from Room_10 to Room_9
Moving Student_1 from Room_9 to Room_10
Moving Student_1 from Room_10 to Room_9
Moving Student_1 from Room_9 to Room_10
Moving Student_1 from Room_10 to Room_9
Moving Student_1 from Room_9 to Room_10
Room_10 state: sticky
